dynamiq/nodes/retrievers/milvus.py

```diff
@@ -93,10 +101,14 @@ def execute(self, input_data: dict[str, Any], config: RunnableConfig = None, **k
         self.run_on_node_execute_run(config.callbacks, **kwargs)
 
         query_embedding = input_data["embedding"]
+        content_key = input_data.get("content_key", None)
```

Comment:
It seems we also should fallback to node configuration:
`content_key = input_data.get("content_key", self.content_key)`
Or I missed smth?